html2text/__init__.py

```diff
@@ -514,6 +514,8 @@ def handle_tag(self, tag, attrs, start):
             else:
                 if self.list:
                     self.list.pop()
+                    if (not self.google_doc) and (not self.list):
```

Comment:
Google Docs implement nested lists using styles. Thus a nested list is actually 3 lists with different formatting.

We could work around that but I do not think they deserve it since they are not following HTML guidelines.